plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/RocksDBTransaction.java

```diff
@@ -37,6 +39,7 @@ public class RocksDBTransaction implements SegmentedKeyValueStorageTransaction {
   private final Transaction innerTx;
   private final WriteOptions options;
   private final Function<SegmentIdentifier, ColumnFamilyHandle> columnFamilyMapper;
+  private final ReadOptions readOptions = new ReadOptions().setVerifyChecksums(false);
```

Comment:
Why is this needed?

Reply:
The get method I added to the transaction requires readOptions. So added it with the same configuration that are being used for `RocksDBColumnarKeyValueStorage`